csharp/private/macros/setup_basic_nuget_package.bzl

```diff
@@ -0,0 +1,94 @@
+load("//csharp/private:providers.bzl", "CSharpAssembly")
+load("//csharp/private:rules/imports.bzl", "import_library", "import_multiframework_library")
+
+def _import_dll(dll, has_pdb, imports):
+    path = dll.split("/")
+
+    tfm = path[1]
+
+    # Ignore frameworks we don't support (like net35)
```

Comment:
I don't know why we don't support net35, lol. It's provided by the NUnit package which is partly why it got handled here.

Reply:
Microsoft doesn't provide a ReferenceAssemblies package for net35.

Reply:
Ah, that would explain it.